tools/depscan/go.mod

```diff
@@ -0,0 +1,17 @@
+module anthoscli/vulnscan
```

Comment:
nit: copy paste

Go introduced db for vulnerabilities recently https://pkg.go.dev/golang.org/x/vulndb

So this looks very aligned with that in approach.

Reply:
Fixed - and removed remnants vuln scanning because go vuln checker is much better there!